Description: keep md files ascii
 It is simpler to generate documentation if only ascii is needed,
 otherwise `ronn` requires to install some UTF8 locale.
Author: Jérémy Lal <kapouer@melix.org>
Forwarded: not-needed
Last-Update: 2014-07-29
--- a/doc/cli/npm-ls.md
+++ b/doc/cli/npm-ls.md
@@ -19,8 +19,8 @@
 For example, running `npm ls promzard` in npm's source tree will show:
 
     npm@@VERSION@ /path/to/npm
-    └─┬ init-package-json@0.0.4
-      └── promzard@0.1.5
+    |-- init-package-json@0.0.4
+      |-- promzard@0.1.5
 
 It will print out extraneous, missing, and invalid packages.
 
--- a/doc/files/package.json.md
+++ b/doc/files/package.json.md
@@ -427,8 +427,8 @@
 installed if needed. `npm install tea-latte` could possibly yield the following
 dependency graph:
 
-    ├── tea-latte@1.3.5
-    └── tea@2.2.0
+    |-- tea-latte@1.3.5
+    |-- tea@2.2.0
 
 Trying to install another plugin with a conflicting requirement will cause an
 error. For this reason, make sure your plugin requirement is as broad as
